tool/update_flutter_sdk.sh

```diff
@@ -24,3 +24,33 @@ else
   ./bin/flutter --version
   popd
 fi
+
+flutter precache --web
+
+# Provision the canvaskit assets.
+
+# This avoids requiring an internet connection for CanvasKit at runtime. This
+# URL should be updated to keep in sync with the version from the engine. See
+# https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/canvaskit/initialization.dart#L50-L78,
+# but compare with the code in master for getting the current version. A better
+# solution would be to either upstream this functionality into the
+# flutter_tools, (https://github.com/flutter/flutter/issues/70101), or to read
+# this from a manifest provided (https://github.com/flutter/flutter/issues/74934).
+
+canvaskit_url=https://unpkg.com/canvaskit-wasm@0.28.1/bin/
+
+flutter_bin=$(which flutter)
+canvaskit_dart_file=$(dirname $flutter_bin)/cache/flutter_web_sdk/lib/_engine/engine/canvaskit/initialization.dart
+if ! grep -q "defaultValue: '$canvaskit_url'" "$canvaskit_dart_file"; then
+  echo "CanvasKit $canvaskit_url does not match local web engine copy. Please update before continuing."
```

Comment:
can we add what the local web engine copy version is to this cli output? This has been a pain point when doing the release in the past. Or at least output a command to easily grep and print the correct version use?

Reply:
Sure, I'll add some cli output here.